.github/workflows/ci.yml

```diff
@@ -8,6 +8,11 @@ on:
       - 'maintainers/**'
   workflow_dispatch:
 
+# Checks if any concurrent jobs under the same pull request or branch are being executed
```

Comment:
See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency